### ISSUE TITLE
History: option to only add views to watched history after watching them for x seconds

### ISSUE THREAD

**Issue by belleDESiRE** [yuliskov/SmartTube#2237]
When I am browsing through a playlist I often skip files pretty quickly that I am not interested in to watch. 
But, they are added to my viewing history almost immediately anyways, it would be nice if that only happens when I have at least watched for a number of seconds (and the amount being configurable, something like, 1,2,3,5,10,20,30)

**Comment by tastyratz** [comment]
I also want to see this. I was just going to add a request to remove videos from history with less than 5 or 10 seconds of playback.

**Comment by asandhu3** [comment]
Agree, history is getting very polluted just from browsing through thumbnails.

**Comment by tastyratz** [comment]
Can these be re-opened without fully resubmitting by replying to the issue?
